app/src/main/res/values/strings.xml

```diff
@@ -79,4 +79,15 @@
     <string name="lat_std_dev_threshold" translatable="false">Latitude Standard Dev Threshold</string>
     <string name="add_note_to_tree">Add note to tree</string>
     <string name="note">NOTE</string>
+
+    <!--Strings for the permission management and rationale-->
+    <string name="accept_camera_permission_header">Accept Camera permission</string>
+    <string name="accept_camera_permission_message">Camera is needed for this app to allow it capture pictures, accept this permission to use this device to take pictures</string>
+    <string name="accept_permission">Accept permission</string>
+    <string name="accept_location_permission_header">Accept Location permission</string>
+    <string name="accept_location_permission_message">Location is needed for this app to allow it track trees, Accept this permission to use the tree tracking feature</string>
+    <string name="open_settings_header">Accept Permissions</string>
+    <string name="open_settings_message">You have denied this permission previously and would have to accept the permissions from the app settings. Click open settings below to accept the permissions required.</string>
```

Comment:
ditto above; maybe something like "Permission currently denied. Click "Open Settings" below to accept the required permissions." ?